apps/next/src/pages/nft/[chainName]/[contractAddress]/[tokenId]/index.tsx

```diff
@@ -20,12 +20,18 @@ export async function getServerSideProps(context) {
     };
 
     const nft = res.data?.data?.item as NFT;
-    const imageUrl = getMediaUrl({
+    let imageUrl = getMediaUrl({
       nft,
       stillPreview: nft?.mime_type?.startsWith("video"),
       optimized: true,
     });
 
+    // lets check if the image is from showtime.xyz (eg Bunny,
+    // since they start with media.showtime.xyz and video.showtime.xyz)
+    if (imageUrl && imageUrl.includes("showtime.xyz/")) {
+      imageUrl = imageUrl + "?class=ogimage";
+    }
+
```

Comment:
This is needed for now, since we still have 3 different mixed sources of images. This can be removed once we have migrated all existing data to Bunny.